src/brsTypes/components/RoString.ts

```diff
@@ -10,16 +10,19 @@ import { Float } from "../Float";
 
 export class RoString extends BrsComponent implements BrsValue, Comparable, Unboxable {
     readonly kind = ValueKind.Object;
-    private intrinsic: BrsString;
+    private intrinsic: BrsString = new BrsString("");
 
     public getValue(): string {
         return this.intrinsic.value;
     }
 
-    constructor(initialValue: BrsString) {
+    constructor(initialValue?: BrsString) {
         super("roString");
 
-        this.intrinsic = initialValue;
+        if (initialValue instanceof BrsString) {
```

Comment:
We probably don't need an `instanceof` check here since we've declared that parameter as `BrsString | undefined` 😃 

```suggestion
        if (initialValue) {
        // or if you want to be explicit:
        // if (initialValue != null) {
```

Reply:
Good call!